src/commands/createNewProject/NewProjectLanguageStep.ts

```diff
@@ -84,7 +85,10 @@ export class NewProjectLanguageStep extends AzureWizardPromptStep<IProjectWizard
                 executeSteps.push(await DotnetProjectCreateStep.createStep(context));
                 break;
             case ProjectLanguage.Python:
-                addPythonCreateProjectSteps(context, promptSteps, executeSteps);
+                const createPythonVenv: boolean = !!getWorkspaceSetting<boolean>('createPythonVenv', context.workspacePath);
+                context.telemetry.properties.createPythonVenv = String(createPythonVenv);
```

Comment:
Haha so close 😂I meant put the telemetry line in `addPythonInitVSCodeSteps.ts` where we're already calling `getWorkspaceSetting`

Reply:
Haha! That makes more sense ;-). Let's give it another try!